docs/runtimes/databricks.ipynb

```diff
@@ -93,6 +93,129 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from dotenv import load_dotenv\n",
+    "import mlrun\n",
+    "from mlrun.runtimes.function_reference import FunctionReference"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile ./databricks_job.py\n",
+    "import os\n",
+    "import random\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from pyspark.sql import SparkSession\n",
+    "\n",
+    "def main():\n",
+    "    df = pd.DataFrame({'A': np.random.randint(1, 100, 5), 'B': np.random.rand(5)})\n",
+    "    path = \"/dbfs/path/folder\"\n",
+    "    parquet_df_path = f\"{path}/df.parquet\"\n",
+    "    csv_df_path = f\"{path}/df.csv\"\n",
+    "    \n",
+    "    if not os.path.exists(path):\n",
+    "        os.makedirs(path)\n",
+    "        \n",
+    "    # save df\n",
+    "    df.to_parquet(parquet_df_path)\n",
+    "    df.to_csv(csv_df_path, index=False)\n",
+    "    \n",
+    "    # log artifact\n",
+    "    mlrun_log_artifact(\"parquet_artifact\",parquet_df_path)\n",
+    "    mlrun_log_artifact(\"csv_artifact\",csv_df_path)\n",
+    "    \n",
+    "    # spark\n",
+    "    spark = SparkSession.builder.appName(\"example\").getOrCreate()\n",
+    "    spark_df = spark.createDataFrame(df)\n",
+    "    \n",
+    "    # spark path format:\n",
+    "    spark_parquet_path = \"dbfs:///path/folder/spark_df.parquet\"\n",
+    "    spark_df.write.mode(\"overwrite\").parquet(spark_parquet_path)\n",
+    "    mlrun_log_artifact(\"spark_artifact\", spark_parquet_path)\n",
+    "    \n",
+    "    #  an illegal artifact does not raise an error, it logs an error log instead."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load env\n",
+    "# Load environment variables from .env file\n",
+    "load_dotenv(\"./credentials.env\")\n",
+    "# The credentials must include DATABRICKS_HOST,DATABRICKS_CLUSTER_ID,DATABRICKS_TOKEN"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def add_databricks_env(project, function):\n",
+    "    job_env = {\n",
+    "        \"DATABRICKS_HOST\": os.environ[\"DATABRICKS_HOST\"],\n",
+    "        \"DATABRICKS_CLUSTER_ID\": os.environ.get(\"DATABRICKS_CLUSTER_ID\"),\n",
+    "    }\n",
+    "    secrets = {\"DATABRICKS_TOKEN\": os.environ[\"DATABRICKS_TOKEN\"]}\n",
+    "\n",
+    "    project.set_secrets(secrets)\n",
+    "    for name, val in job_env.items():\n",
+    "        function.spec.env.append({\"name\": name, \"value\": val})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "project_name = \"databricks-log-artifact\"\n",
```

Comment:
I want to use the same object as before...
so you can delete this two lines:

project_name=...
project=...

Reply:
deleted